Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/EndOfYearDataManager.swift

```diff
@@ -407,6 +434,20 @@ public struct YearOverYearListeningTime {
     }
 }
 
+public struct EpisodesStartedAndCompleted {
+    public let started: Int
+    public let completed: Int
+
+    public var percentage: Double {
+        Double(completed) / Double(started)
+    }
+
+    public init(started: Int, completed: Int) {
+        self.started = started
+        self.completed = completed
+    }
```

Comment:
Wdyt about:
- storing the computer the percent in init
- Clamping the percent to between 0 and 1
- If, for some reason the completed number if higher than the started, then use the completed number as the started. 

```suggestion
    public let percentage: Double
    
    public init(started: Int, completed: Int) {
        self.started = max(started, completed)
        self.completed = completed
        self.percentage = (Double(completed) / Double(started)).clamped(to: 0..<1)
    }
```